ourprojects/models.py

```diff
@@ -30,20 +70,116 @@ class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
 
     id: int = models.AutoField(primary_key=True)
     """Internal id, valid only in one DB instance."""
-    uid: str = CharField(unique=True, max_length=12, default=ids.base62_12)
+    uid: str = CharField(
+        unique=True, max_length=12, db_index=True, default=ids.base62_12
+    )
     """Universal id, valid across DB instances."""
     name: str = CharField(db_index=True)
     """Title or name of the Project."""
     abbr: str | None = CharField(max_length=32, db_index=True, unique=True, null=True)
     """A unique abbreviation."""
     url: str | None = URLField(max_length=255, null=True, default=None)
     """A URL to view."""
+    people: Person = models.ManyToManyField(Person, related_name="project_people")
+    """People associated with this project."""
+    references: Reference = models.ManyToManyField(
+        "Reference", related_name="project_references"
+    )
+    """References associated with this project."""
     artifacts: Artifact = models.ManyToManyField(
         Artifact, through="ArtifactProject", related_name="Projects"
     )
```

Comment:
Is there a reason the 3 related_name here are different from our convention?

They should all be `projects` (lower case) I think.